spa/src/components/pageEditor/PageEditor.js

```diff
@@ -377,34 +378,53 @@ function PageEditor() {
                   buttonType="neutral"
                   color={theme.colors.primary}
                   data-testid="preview-page-button"
+                  toolTiptext="View"
```

Comment:
CamelCase was swapped. `tooltip` + `Text`
```suggestion
                  tooltipText="View"
```